### PR TITLE
Screener tests for sticky breadcrumb and details list header

### PR DESCRIPTION
**Pull request by tharshada**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Adding screener tests for a scrollable DetailsList with sticky breadcrumb and column headers. The test scrolls the DetailsList to various positions and verifies that the sticky header is still positioned as expected.

Related change: https://github.com/OfficeDev/office-ui-fabric-react/pull/6937/files

Expected:
image

Screenshot from when this was broken in the past (ignore the theme):
image

[More info](https://microsoft-my.sharepoint.com/:f:/p/hatrived/ErYBJRml-mJNoevBTyv7nAQBJHMHWMMIg_2Ig8vLxfV3Og?e=LVMg44)

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7595)

### CODE REVIEW

**Comment by ecraig12345** [comment]
Thanks for adding these tests @tharshada! Would you mind updating the PR description itself with some more detail on the scenario and the tests you added?

**Comment by tharshada** [comment]
> Thanks for adding these tests @tharshada! Would you mind updating the PR description itself with some more detail on the scenario and the tests you added? For an open source project, it's best to include all the relevant details publicly instead of linking to content in your OneDrive that may not be accessible to everyone and may be deleted later.

@ecraig12345 updated the description

File: apps/vr-tests/src/stories/Sticky.Breadcrumb.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    <Screener
+      steps={new Screener.Steps()
+        .snapshot('default', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollTop = 2")
+        .snapshot('scroll down by a small amount so that the first row is still visible and the header becomes sticky', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollTop = 99999")
+        .snapshot('scroll down to the bottom', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollTop = 0")
+        .snapshot('scroll up to the top', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollLeft = 10")
```
Comment (ecraig12345) [inline apps/vr-tests/src/stories/Sticky.Breadcrumb.stories.tsx]: Might be good to scroll left by a larger amount--10px is so small that it's hard to see in the screener snapshot.

image

Comment (KevinTCoughlin) [reply apps/vr-tests/src/stories/Sticky.Breadcrumb.stories.tsx]: Agreed -- Was going to comment that some of these tests which scroll traverse an amount that is barely noticeable. +1 to increasing the distance.

File: apps/vr-tests/src/stories/Sticky.Breadcrumb.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        .snapshot('scroll left', { cropTo: '.testWrapper' })
```
Comment (KevinTCoughlin) [inline apps/vr-tests/src/stories/Sticky.Breadcrumb.stories.tsx]: Isn't this test scrolling to the "right" in LTR? Confused me when looking at the screener output.

**Comment by dzearing** [comment]
2 of the screenshots look a little off to me; the left margin disappeared. Can you investigate and confirm if expected?

**Comment by ecraig12345** [comment]
I thought the point of this screenshot and the similar one was that they're slightly scrolled to the right. @tharshada Can you confirm? (And it would be easier to see that the view is scrolled if you make the scroll amount larger than 10px, like Kevin and I suggested.)

image

**Comment by dzearing** [comment]
This doesn't quite make sense to me:

image

I would expect if you scroll horizontally, the details list would be scrolled but not the breadcrumb. Why is it the opposite?

**Comment by tharshada** [comment]
> This doesn't quite make sense to me:
> 
> image
> 
> I would expect if you scroll horizontally, the details list would be scrolled but not the breadcrumb. Why is it the opposite?

In non-sticky state, breadcrumb is in ms-ScrollablePane--contentContainer. When it becomes sticky,
it is in stickyAbove container. Here is a similar [example](https://fabricweb.z5.web.core.windows.net/oufr/6.49.0/#/examples/scrollablepane)

**Comment by dzearing** [comment]
I see now. Thank you @tharshada for your patience :)

**Pull request merged**